github-release

```diff
@@ -170,7 +170,18 @@ REPOSITORY=$(git config --get remote.origin.url | sed -E 's/https?:\/\/(www.)?gi
 GREY_COLOR="\033[0;37m"
 NO_COLOR="\033[0m"
 
-echo -e "Detected remote repository ${GREY_COLOR}${REPOSITORY}${NO_COLOR}."
+if [ ! -z "$REPOSITORY" ]
+  then
+    echo -e "Detected remote repository ${GREY_COLOR}${REPOSITORY}${NO_COLOR}."
+  else
+    echo -e "Remote ${GREY_COLOR}origin${NO_COLOR} missing"
+    echo -e "You need to specify the remote repository manually"
+    read -r -p 'Repository URL > ' REPO_URL
+    read -r -p 'Default Branch > ' REPO_BRANCH
+    echo -e "Adding specified remote as ${GREY_COLOR}origin${NO_COLOR}"
+    git remote add origin "${REPO_URL}" -m "${REPO_BRANCH}"
+    echo -e "Done"
```

Comment:
This is great, neat prompt! But currently it is not working correctly because the `$REPOSITORY` is not set after adding the new remote. I see a couple options:

* Copy the line that reads the repository handle down into this case after the new remote is added.
* Make the line that reads the repository handle a function and call it in the two places.
* Somehow check if remote origin exists and is a GitHub repo first, ask for it if not. And then extract the repository handle only once after that.

All of them are fine with me 👍

Reply:
2nd option is what I thought of doing. Would do now
3rd option, Yes, I specified it could be done on request

Reply:
So I'd suggest going with the second option.